wom/src/main/scala/wom/expression/WomExpression.scala

```diff
@@ -36,3 +37,21 @@ trait IoFunctionSet {
   def glob(pattern: String): Seq[String]
   def size(params: Seq[Try[WomValue]]): Try[WomFloat]
 }
+
+/**
+  * Simply looks up the id in the inputs map.
+  */
+case class InputLookupExpression(tpe: WomType, id: String) extends WomExpression {
+
+  override def sourceString: String = id
+
+  override def evaluateValue(inputValues: Map[String, WomValue], ioFunctionSet: IoFunctionSet): ErrorOr[WomValue] =
+    inputValues.get(id).toValidNel(s"could not find $id in $inputValues!")
+
+  override def evaluateType(inputTypes: Map[String, WomType]): ErrorOr[WomType] = validNel(tpe)
+
+  override def evaluateFiles(inputTypes: Map[String, WomValue], ioFunctionSet: IoFunctionSet, coerceTo: WomType): ErrorOr[Set[WomFile]] =
+    validNel(Set.empty)
```

Comment:
Is this necessarily true ?

Reply:
I'll take another look.  I keep forgetting what `evaluateFiles` is doing without looking at Cromwell, I think I'll make the name more descriptive.